### PR TITLE
Fix multiple cves

### PR DESCRIPTION
**Pull request by andir**
###### Motivation for this change

I created a new tool to generate a list of packages affected by CVEs and started working through the list for the unstable channel.
This is just the ones that seemed fixable and where proper upstream commits/patches could be found.
The complete list from my WIP tool can be found [here](https://epsilon.rammhold.de/andi/nix-updates/).

Some of the commits might be worth backporting to stable.

- e8a3ce0 busybox
- 0b4e8b9 libexif
- f8b53a7 redis
- 4b759a0 rzip
- e15d6e1 yara (minor version bump)
- 8312eaf radare2 (minor version bump + patch)

I am not so sure about the others since they might involve bumping versions. Is that something we do for an already released stable version?

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by vcunat** [comment]
Yet another tool to list CVEs for our packages :-)

**Comment by andir** [comment]
Not just CVEs, also "upstream" version changes from e.g. release-monitoring.org... :-)

**Comment by vcunat** [comment]
That reminds me of https://github.com/Phreedom/nixpkgs-monitor

**Comment by copumpkin** [comment]
cc @NixOS/security-notifications on both the specific fixes and the tool

File: pkgs/tools/backup/partclone/default.nix [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
   name = "partclone-${version}";
-  version = "0.2.89";
+  version = "0.3.11";
 
   src = fetchFromGitHub {
     owner = "Thomas-Tsai";
     repo = "partclone";
     rev = version;
-    sha256 = "0gw47pchqshhm00yf34qgxh6bh2jfryv0sm7ghwn77bv5gzwr481";
+    sha256 = "1gw47pchqshhm00yf34qgxh6bh2jfryv0sm7ghwn77bv5gzwr481";
```
Comment (orivej) [inline pkgs/tools/backup/partclone/default.nix]: This sha256 is invalid, it should be `0bv15i0gxym4dv48rgaavh8p94waryn1l6viis6qh5zm9cd08skg`.

**Review by vcunat — COMMENTED** [review]
OK, with some additional changes.

**Comment by vcunat** [comment]
Merged as cbfb586151c with some changes in 6255e95a3d, ac677c7a017c and 73bec9767.

For partclone I could find no reasonable news/announcement (it at least has no reverse dependencies in nixpkgs);  the rest seems safe for cherry-picking to me.

**Comment by fpletz** [comment]
Thank you for working on and merging this!

**Pull request closed without merging**